docs/supported-backends.md

```diff
@@ -0,0 +1,10 @@
+# Supported Backends
+
+Currently, Tracetest supports the following backend datastores:
+
+- Jaeger
+- Grafana Tempo
+
+Details of configuring Tracetest to access these is discussed in the [installation instructions](/docs/advanced-selectors.md).
```

Comment:
is `/docs/advanced-selectors.md` correct here?

Reply:
Yes - thanks - changes made